services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -113,6 +119,50 @@ class HankeServiceITests : DatabaseTest() {
         confirmVerified(cableReportService)
     }
 
+    @Nested
+    inner class GetHankeId {
```

Comment:
Since we care about code coverage, we should have tests also for these pretty trivial service methods. 

What I'm not sure about is whether it's better to test these with integration tests like this or with unit tests like in HankeServiceTest. With more complicated methods I prefer integration tests since they test more, but here it feels like the integration test is mainly testing Spring Data implementation.